### DISCUSSION TITLE
Unable to verify jwt token with email_passwordless login (custom authentication)

### DISCUSSION THREAD

**Discussion by NicoBeskar** [Web3Auth/Web3Auth#514] [Help]
I'm having trouble verifying authentication with email_passwordless (custom authentication). In my verifier I am loading my client ID and domain from auth0 correctly, but when the authentication process finishes I get this error:

Could not get result from torus nodes Error occurred while verifying paramsunable to verify jwt token, [failed to verify jws signature: failed to verify message: crypto/rsa: verification error failed to verify jws signature: failed to verify message: crypto/rsa: verification error]

What would be the correct way to verify authentication with email_passwordless?

**Comment by shahbaz17** [comment]
Hey @NicoBeskar 

Can you share the code snippet of initialisation and login?

Also, have you looked into [Auth0 Guide](https://web3auth.io/docs/guides/auth0)? It might help you know how the login works and with the code snippets.

**Reply by NicoBeskar** [reply]
Initialization:

```
const openloginAdapter = new OpenloginAdapter({
	adapterSettings: {
		clientId: clientId,
		network: "testnet",
		uxMode: "popup",
		loginConfig: {
			google: {
				name: "Beru Google Login",
				verifier: "beskar-google-verifier",
				typeOfLogin: "google",
				clientId: "my GOOGLE_CLIENT_ID"
			},
			facebook: {
				name: "Beru Facebook Login",
				verifier: "beskar-facebook-verifier",
				typeOfLogin: "facebook",
				clientId: "my FACEBOOK_CLIENT_ID"
			},
			jwt: {
				name: "Beru Twitter Login",
				verifier: "beskar-twitter-verifier",
				typeOfLogin: "twitter",
				clientId: "my TWITTER_CLIENT_ID"
			},
			email_passwordless: {
				name: 'Beru Email Login',
				verifier: "beskar-emailpassword-verifier",
				typeOfLogin: "email_password",
				clientId: "my auth0 client id"
			}
		},
	},
});

this.web3Auth.configureAdapter(openloginAdapter);
this.web3Auth.init();
```



Login:


```
this.web3Auth.connectTo(WALLET_ADAPTERS.OPENLOGIN, {
	loginProvider: 'email_passwordless',
	extraLoginOptions: {
		login_hint: email
	}
})
```

**Comment by yashovardhan** [comment]
Hey @NicoBeskar 

As I see, your login code looks incorrect. Please enter the domain and sub fields from auth0 here as well if you're using the SPA flow. You can checkout this guide for help on how to configure your auth0 login with web3auth: https://web3auth.io/docs/guides/auth0#logging-in
